From 8a3e6696724474ebc175a1b02ba1f96f436a4abb Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Mon, 6 Jan 2020 20:26:31 +0100
Subject: [PATCH] Fix error on creating warning message dialog for error
 scanning page

SimpleDialog constructor requires to provide the parent window.
---
 src/ocrfeeder/studio/studioBuilder.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ocrfeeder/studio/studioBuilder.py b/src/ocrfeeder/studio/studioBuilder.py
index f713c7c..7a2ccdc 100644
--- a/src/ocrfeeder/studio/studioBuilder.py
+++ b/src/ocrfeeder/studio/studioBuilder.py
@@ -231,7 +231,7 @@ class Studio:
             self.__addImagesToReviewer([image_path])
             Gdk.threads_leave()
         else:
-            error = widgetPresenter.SimpleDialog(\
+            error = widgetPresenter.SimpleDialog(self.main_window,
                                              _("Error scanning page"),
                                              _("Error"),
                                              'warning')
-- 
2.24.1

